### ISSUE TITLE
graph view issues

### ISSUE THREAD

**Issue by helenp** [EBISPOT/OLS#114]
I want to visualise this GO term to use in a ppt http://www.ebi.ac.uk/ols/ontologies/go/terms/graph?iri=http://purl.obolibrary.org/obo/GO_0044461

But, when I move leaves one by one to get them non-overlapping the graph automatically re-organises and puts them on top of each other again. This makes me think the library we are using is suboptimal.

**Comment by LLTommy** [comment]
Find the button "Turn of physics" below the graph. If psychics are turned off, you can drag the nodes around without them reorganising themselves.

**Comment by helenp** [comment]
1. That is a rubbish label - physics of the graph -> turn of autoarrange?
2. Consequences if turn it off by default? - induces nausea and never seen a layout I liked

**Comment by LLTommy** [comment]
1. We did not think about the label too much quite frankly but I agree, it might not be a good name. We can change that.
2. To place the nodes in first place, you need some sort of rule. These are force based physics engines usually and that is what is used. So we need it to arrange the nodes at start up, I could turn it of by default once the layout is computed (e.g. invert the button functionality)

**Comment by jmcmurry** [comment]
I agree with @helenp about the label being unhelpful. You might consider a labeled toggle between manual and auto arrangement? 

<img width="200" alt="screen shot 2016-09-20 at 9 49 30 am" src="https://cloud.githubusercontent.com/assets/3658863/18680264/d3c81796-7f17-11e6-92a2-06a89b31b3e2.png">

Personally, I find these a tiny bit more intuitive than "on / off" switches but don't feel very strongly about it. A better label would go a long way already.

ps. I'm loving the psychics typo :)
> If psychics are turned off, you can drag the nodes around without them reorganising themselves.

**Comment by LLTommy** [comment]
Agreed, the label is not the best. Will be changed. I explained why it is called that way - because it is based on physics engine in the background - but if you don't know that or don't think about it, the label makes no sense. 

I don't know if that was a typo or a typo in combination with autocorrection, but in any case, yes I replied while I was listening to a talk - a perfect example of not successful multitasking :)
